### PR TITLE
Sprint 25 gw

### PR DESCRIPTION
**Pull request by G-White-ISB**
Fixes display of cohorts (re github issue 653). I had this done a while ago but did not push it.

### CODE REVIEW

File: solr_helpers/__init__.py [hunk 1]
```diff
@@ -556,7 +558,9 @@ def build_solr_query(filters, comb_with='OR', with_tags_for_ex=False, subq_join_
             count += 1
 
         query_set[attr_name] = query_str
-
+    #filter_tags ==None in IDC
+    i=1
+    pass
```
Comment (s-paquette) [inline solr_helpers/__init__.py:563]: I'm unclear what this is meant to accomplish? We definitely tag filters in IDC--that's how we do exclusions for counting purposes.

**Review by s-paquette — CHANGES_REQUESTED** [review]

File: solr_helpers/__init__.py [hunk 2]
```diff
@@ -403,7 +405,7 @@ def build_solr_query(filters, comb_with='OR', with_tags_for_ex=False, subq_join_
             mutation_filters[attr] = values
         else:
             main_filters[attr] = values
-
+    #ISB-CGC legacy??? Is this applicable in IDC??? Might lead to problems
     # Mutation filters
```
Comment (s-paquette) [inline solr_helpers/__init__.py:409]: Yes, these are going to be implemented. What problems are you anticipating?

File: solr_helpers/__init__.py [hunk 3]
```diff
@@ -384,6 +384,8 @@ def build_solr_facets(attrs, filter_tags=None, include_nulls=True, unique=None,
 #
 def build_solr_query(filters, comb_with='OR', with_tags_for_ex=False, subq_join_field=None,
                      search_child_records_by=None):
+
+    # subq_join seems to be unused in IDC??
     ranged_attrs = Attribute.get_ranged_attrs()
```
Comment (s-paquette) [inline solr_helpers/__init__.py:389]: It isn't for the moment due to collapsed record style, can't guarantee it never will.

File: solr_helpers/__init__.py (hunk 1 above)
Comment (G-White-ISB) [reply solr_helpers/__init__.py]: I just had this in as breakpoints

File: solr_helpers/__init__.py (hunk 2 above)
Comment (G-White-ISB) [reply solr_helpers/__init__.py]: Right now we do not have any 'mutation' filters so this block is not being used. If we had an attribute that happened to end with 'MUT' could that mess things up

**Review by s-paquette — APPROVED** [review]

**Pull request merged**